aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/ContainerDrivenLayout.js

```diff
@@ -9,8 +15,9 @@ import {
   ThemeContext,
 } from 'grommet';
 
+import { FormDown, FormNext } from 'grommet-icons';
```

Comment:
Per brand refresh, we are migrating away from use of the "Form" version of icons.

```suggestion
import { Down, Next } from 'grommet-icons';
```